client/pdf_generator.py

```diff
@@ -51,21 +48,20 @@ class Plugin(plugins.ClientPlugin):
 		),
 		plugins.ClientOptionString(
 			'logo',
-			'Image to include into the pdf',
-			display_name="Logo Image"
+			'platypus.Image to include into the pdf',
```

Comment:
did you purposely do a platupus.Image here or was this a result of a find and replace?